### DISCUSSION TITLE
Is eslint:source-code-fixer run, even when there is no --fix flag?

### DISCUSSION THREAD

**Discussion by johannesjo** [eslint/eslint#15105] [Q&A] [answered]
When running eslint in debug mode for [my project](https://github.com/johannesjo/super-productivity) I get the following output for each item (with slightly different values for the time)
```
  eslint:source-code-fixer Applying fixes +5s
  eslint:source-code-fixer shouldFix parameter was false, not attempting fixes +0ms
```

The Applying fixes part seems to take a significant amount of time, even though they're not applied in the end. Am I doing something wrong? Or is just the output maybe misleading?

**Comment by nzakas** [comment, marked as answer]
I think you are misreading the output. The function is called to apply fixes but then exits without doing so. When in doubt, check the source: https://github.com/eslint/eslint/blob/a79c9f35d665c2bcc63267bdf359a8176e0a84ce/lib/linter/source-code-fixer.js#L61-L71

**Reply by johannesjo** [reply]
Thanks for the link! I still wonder what the +5s means then?

**Reply by mdjermanovic** [reply]
`+5s` is time diff between that log and the previous log with the same namespace `eslint:source-code-fixer`.

**Reply by johannesjo** [reply]
Ah alright! Thanks for the explanation!

**Reply by LaytonGB** [reply]
This is very misleading! Should the message perhaps be changed?
